packages/fast-components-react-base/src/card/card.props.ts

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import { ICardClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+
+/**
+ * Card HTML tags
+ */
+export enum CardHTMLTags {
+    article = "article",
+    div = "div",
+    section = "section"
+}
+
+export interface ICardUnhandledProps extends React.HTMLAttributes<HTMLDivElement | HTMLElement> {}
+export interface ICardManagedClasses extends IManagedClasses<ICardClassNameContract> {}
+export interface ICardHandledProps {
+    /**
+     * The card children
+     */
+    children?: React.ReactNode;
```

Comment:
I believe Foundation already has children specified as a handledProp

Reply:
We enumerate in the prop contract because it's possible certain components may not take children (text field base component), but we don't have to enumerate as part of the handledProp value inside the class for Card because if children are passed we never want to pass it to the DOM.